eel/tests/persistence_test.rs

```diff
@@ -57,7 +67,27 @@ mod persistence_test {
             .contains("RemoteStorageServiceUnavailable"));
     }
 
-    fn run_flow(node_handle: &NodeHandle) {
+    fn run_flow_normal_restart(node_handle: &NodeHandle) {
+        run_flow_1(node_handle);
+
+        // Wait for shutdown to complete
+        sleep(Duration::from_secs(5));
+
+        run_flow_2(node_handle);
+    }
+
+    fn run_flow_recovery_restart(node_handle: &NodeHandle) {
+        run_flow_1(node_handle);
+
+        // Wait for shutdown to complete
+        sleep(Duration::from_secs(5));
+        // Remove the local state
+        fs::remove_dir_all(".3l_local_test").unwrap();
+
+        run_flow_2(node_handle);
+    }
+
+    fn run_flow_1(node_handle: &NodeHandle) {
```

Comment:
I'm not sure what exactly `run_flow_1` and `run_flow_2` does, but it looks to me like they could be renamed to something like `run_flow_1st_jit_channel` and `run_flow_2nd_jit_channel`

Reply:
[8c9a5a3](https://github.com/getlipa/lipa-lightning-lib/pull/204/commits/8c9a5a374e16bdffeb6aa3dbe9255fb8af275c8e)